benchmarks/ddtrace_run/scenario.py

```diff
@@ -6,13 +6,14 @@
 
 
 class DDtraceRun(bm.Scenario):
-    ddtrace_run = bm.var_bool()
-    http = bm.var_bool()
-    runtimemetrics = bm.var_bool()
-    telemetry = bm.var_bool()
-    profiling = bm.var_bool()
-    appsec = bm.var_bool()
-    tracing = bm.var_bool()
+    name: str
```

Comment:
nit: Would it make sense to have this defined in `bm.Scenario`, assuming all subclasses of it will have a name?

Reply:
Actually taking this back, seeing a lot of subclasses without `name`. Unless it makes sense to add that field to the others, ignore this comment 😅

Reply:
Oh, this one is left over from a previous version where the subclasses had the `name` instead of `bm.Scenario`, it's already there so I'll remove it from here